WalletWasabi/Legal/Assets/LegalDocumentsGingerWallet.txt

```diff
@@ -0,0 +1,484 @@
+
+
+=========================================
+
+SUMMARY
+
+=========================================
+
+
+    The GingerPrivacy team is committed to being fully transparent to its users, from its source code to the legal aspects of its services. The purpose of Ginger Wallet is to provide privacy for its users’ Bitcoin transactions. This means that with the use of the wallet, we would like to enable the users to make Bitcoin payments in such a way that their spending history, wallet balances, etc. are not visible to the receiving party. This does not necessarily mean that the user stays anonymous since the receiving party might have to identify the sending party.
+
+    Our goal is to minimize the unintentional leaking or disclosure of private financial information to third parties and the receiver of any transaction resulting from the use of the Ginger Wallet. The protection of your privacy and your personal data is our highest priority, which also defines the essence of our services.
+
+    One instance of the Ginger Wallet CoinJoin coordinating server is operated by a legal entity, InvisibleBit LLC. Thus, to legally provide our services to our clients, we have defined the framework and the rules that both parties must follow to protect each other.
+
+
+    The most important parts of the document below are summarized in the following points:
+
+     - The service is open-source under the MIT license.
+
+     - The service is provided on a non-custodial basis. Safekeeping of keys is the sole responsibility of the user.
+
+     - The user is solely responsible for acting according to their local laws and regulations.
+
+     - We do not store any personally identifiable information. Moreover, our trustless software architecture prevents us from gathering this information in the first place.
+
+     - A transaction fee is only charged by the Service Provider for CoinJoin transactions.
+
+     - We only provide written support, and NEVER ask for recovery words, passwords, or similar security critical information.
+
+
+    The services are provided between remote parties. Therefore, these Terms and Conditions cannot be negotiated with clients. This document will help our clients to understand exactly the Terms and Conditions under which We can provide Our services. We recommend you first learn Bitcoin basics before using Our Services.
+
+
+    PLEASE READ THE BELOW TERMS AND CONDITIONS CAREFULLY. BY CLICKING AGREE, OR BY ACCESSING OR DOWNLOADING OUR SOFTWARE (AS DEFINED BELOW), YOU AGREE TO BE BOUND BY THESE TERMS AND ALL TERMS INCORPORATED BY REFERENCE.
```

Comment:
Does `by accessing our software` include that if someone else connects to our coordinator with a 3rd party software?
It might be better to mention that explicitly.

```suggestion
    PLEASE READ THE BELOW TERMS AND CONDITIONS CAREFULLY. BY CLICKING AGREE, OR BY ACCESSING OR DOWNLOADING OUR SOFTWARE (AS DEFINED BELOW), YOU AGREE TO BE BOUND BY THESE TERMS AND ALL TERMS INCORPORATED BY REFERENCE.
```

Reply:
Ahh L83 covers it.